src/week1.org

```diff
@@ -0,0 +1,41 @@
+#+TITLE: Week 1
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org
+
+* What are we doing today?
+- Trainer introductions
+- Overview of the RoboJackets software training program
+- Navigating and manipulating files and directories in the Linux terminal
+- Using Git and GitHub for version control
+
+* Who am I?
+- Kristaps Berzinch
+  - RoboJackets Treasurer
+  - Linux Foundation Certified Engineer
+- How to contact me
+  - Slack: @kristapsberzinch
+  - Email: kristaps@robojackets.org
+
+* Why are you here?
+- Learn the basics of software development
+- Get up to speed on technologies and techniques common to most RoboJackets teams
+- TODO
+
+* The Linux terminal
+- Powerful text-based interface for interacting with your computer
+- (Almost) everything is a file
+- There's one big happy file system
+- Lots of tiny tools you can chain together to do useful things
+
+* Basic commands
+| Command | Use |
+|-------+------|
+| =cd= | Change to a different directory |
```

Comment:
I put `pwd` on the list when we originally discussed this and it was removed in the interest of keeping this list short.